automod/triggers.go

```diff
@@ -1155,6 +1208,10 @@ func (nwl *NicknameWordlistTrigger) CheckNickname(t *TriggerContext) (bool, erro
 	}
 
 	fields := strings.Fields(PrepareMessageForWordCheck(t.MS.Member.Nick))
+	if dataCast.SanitizeText {
+		messageFieldsFixTextd := strings.Fields(common.SanitizeText(PrepareMessageForWordCheck(t.MS.Member.Nick)))
```

Comment:
`messageFieldsFixTextd` what does the extra `d` at the end signify in the variable name?

Reply:
That, was an accident. Thanks for picking that up lol